assets/sass/components/_badge.scss

```diff
@@ -0,0 +1,18 @@
+@import "../mixins/badge";
+
+.badge {
+	@include badge-color($aplus-default-status-bg, #000);
+
+	&-success {
```

Comment:
(no change requested here) Concerning the variable names, since A+ uses CSS classes `badge-success` and `badge-danger` etc. everywhere, we still want to keep those class names. We don't want to change them everywhere at this time.